boray/src/bucket/create.rs

```diff
@@ -59,11 +59,7 @@ pub(crate) fn action(
 
             // Database errors are returned to as regular Fast messages
             // to be handled by the calling application
-            let value = array_wrap(json!({
-                "name": "PostgresError",
-                "message": e
-            }));
-
+            let value = sql::postgres_error(e);
             let msg_data = FastMessageData::new(method.into(), value);
```

Comment:
I think we still probably need to call `array_wrap` on `value`. It doesn't necessarily make sense, but Fast demands everything an array. I can test it out to be sure that it would be a problem, but I feel like it might be the case.

Reply:
FastRequestError: request failed: fast protocol: data.d for DATA and END messages must be an array

you're totally right! i'll make the change and test it